File: week_01/exercises/cash.c
#include <stdio.h>
#include <cs50.h>

void calculate_cents(int cents);

int main(void){
    int cents;
    do
    {
        cents = get_int("Change owed: ");
    } while(cents < 0);

    calculate_cents(cents);
}

void calculate_cents(int cents){
    int change = 0;
    while(cents >= 25){
        change++;
        cents -= 25;
    }
    while(cents >= 10){
        change++;
        cents -= 10;
    }
    while(cents >= 5){
        change++;
        cents -= 5;
    }
    while(cents >=  1){
        change++;
        cents -= 1;
    }

    printf("%d\n", change);
}

File: week_06/exercises/cash.py
# ESSA PASTA É PASTA É PARA TESTE DE BIBLIOTECA
def calculate_cents(cents):
    change = 0
    while cents >= 0.25:
        change += 1
        cents -= 0.25
    while cents >= 0.10:
        change += 1
        cents -= 0.10
    while cents >= 0.5:
        change += 1
        cents -= 0.5
    while cents >= 0.01:
        change += 1
        cents -= 0.1
    print(f"{change}", change)

while True:
    try:
        cents = float(input("Change owned: "))
        if cents >= 0:
            break
    except ValueError:
        pass
calculate_cents(cents)

# Calling the function
calculate_cents(cents)